### PR TITLE
Issue - Setup Github Actions #26

### PR DESCRIPTION
**Pull request by ank1traj**
# Pull Request

## Description

This PR will fix the issue (Setup Github Actions #26) and add the GitHub actions

## Related Issue

[Setup Github Actions #26](url)

## Version

Did you remember to update the version number in the package.json file?

- [ ] For a major restructuring, or large changes please bump the the major version number. (ex: 1.x.x)
- [ ] For a minor change, bump the minor version number. (ex: x.1.x)
- [ ] For a bug fix, bump the patch version number. (ex: x.x.1)

## Documentation

For _any_ version updates, please verify if the [documentation page]() needs an update. If it does, please [uptade the in code docs]() to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

File: .github/workflows/first-interaction.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Congrats on creating your first issue and thanks for time to improve Open Source.'
```
Comment (j-dogcoder) [inline .github/workflows/first-interaction.yml:17]: Please change the message to:

`Congratulations on opening your first issue to the project! We will triage this shortly.`

File: .github/workflows/first-interaction.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        pr-message: 'Congrats on making your first Pull Request and thanks for time to improve Open Source.'
```
Comment (j-dogcoder) [inline .github/workflows/first-interaction.yml:18]: Please change the message to:

`Congratulations on opening your first pull request to the project! A core maintainer will review your contribution shortly.`

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "api",
-  "version": "4.1.0",
+  "version": "4.1.1",
```
Comment (j-dogcoder) [inline package.json:3]: this does not get a version update. please change it back to v4.1.0

File: .github/workflows/stale.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```
Comment (j-dogcoder) [inline .github/workflows/stale.yml:17]: Please change the close comment to `true`

The message should say
`This issue is now locked due to inactivity.`

**Review by j-dogcoder — CHANGES_REQUESTED** [review]

**Comment by j-dogcoder** [comment]
Hey @ank1traj! I left you some feedback.

**Comment by ank1traj** [comment]
@j-dogcoder Changed as per feedback

**Review by j-dogcoder — APPROVED** [review]

**Pull request merged**